src/JustSaying/AwsTools/MessageHandling/SnsTopicByName.cs

```diff
@@ -14,8 +14,10 @@ namespace JustSaying.AwsTools.MessageHandling
 {
     public class SnsTopicByName : SnsTopicBase
     {
+        private readonly ILogger _logger;
+
         public string TopicName { get; }
-        private readonly ILogger _log;
+        public Dictionary<string, string> Tags { get; set; }
```

Comment:
Better to use `IDictionary<string, string>` than use the specific concrete type?